src/chunkserver/copyset_node.cpp

```diff
@@ -658,6 +688,12 @@ bool CopysetNode::IsLeaderTerm() const {
     return false;
 }
 
+bool CopysetNode::IsLeaseLeader(const braft::LeaderLeaseStatus &lease_status) const {  // NOLINT
+    auto term = leaderTerm_.load(std::memory_order_acquire);
+    // if term == lease_status.term, the lease status must be LEASE_VALID
+    return term > 0 && term == lease_status.term;
```

Comment:
Why not use lease_status.state==LEASE_VALID directly to judge？

Reply:
Why `const braft::LeaderLeaseStatus &lease_status` needs to be passed into the function as a parameter instead of getting the status directly in the function？